examples/with-localstorage/README.md

```diff
@@ -20,28 +20,6 @@ Storing tokens in localstorage comes with a few drawbacks:
 
 For a detailed comparison please see [our blog](https://supertokens.com/blog/cookies-vs-localstorage-for-sessions-everything-you-need-to-know)
 
```

Comment:
i think this example app can be removed entirely and instead the readme can just point to the docs about token transfer method section.

I know this is localstorage and we store in fronternd cookies, but the point of this example app was to show how users could use with header based auth instead of cookie based.